libs/coin-framework/src/account/support.ts

```diff
@@ -0,0 +1,111 @@
+import {
+  AccountNotSupported,
+  CurrencyNotSupported,
+  UnavailableTezosOriginatedAccountReceive,
+} from "@ledgerhq/errors";
+import { getEnv } from "../env";
+import { decodeAccountId } from "./accountId";
+import {
+  getAllDerivationModes,
+  getDerivationModesForCurrency,
+} from "../derivation";
+import { isCurrencySupported } from "../currencies";
+// import { getMainAccount } from "../account";
+// import { getAccountBridge } from "../bridge";
+import type { CryptoCurrency } from "@ledgerhq/types-cryptoassets";
+import type { Account, AccountLike } from "@ledgerhq/types-live";
+import type { DerivationMode } from "../derivation";
+
+export const shouldShowNewAccount = (
+  currency: CryptoCurrency,
+  derivationMode: DerivationMode
+): boolean => {
+  const modes = getDerivationModesForCurrency(currency);
+  // last mode is always creatable by convention
+  if (modes[modes.length - 1] === derivationMode) return true;
+  // legacy is only available with flag SHOW_LEGACY_NEW_ACCOUNT
+  if (
+    derivationMode === "" &&
+    (!!getEnv("SHOW_LEGACY_NEW_ACCOUNT") || currency.family === "bitcoin")
+  )
+    return true;
+  // native segwit being not yet supported everywhere, segwit is always available for creation
+  if (
+    derivationMode === "segwit" ||
+    (currency.family === "bitcoin" &&
+      (derivationMode === "native_segwit" || derivationMode === "taproot"))
+  )
+    return true;
+  return false;
+};
+export const getReceiveFlowError = (
+  account: AccountLike,
+  parentAccount: Account | null | undefined
+): Error | null | undefined => {
+  if (parentAccount && parentAccount.currency.id === "tezos") {
+    return new UnavailableTezosOriginatedAccountReceive("");
+  }
+};
+// export function canSend(
+//   account: AccountLike,
+//   parentAccount: Account | null | undefined
+// ): boolean {
+//   try {
+//     getAccountBridge(account, parentAccount).createTransaction(
+//       getMainAccount(account, parentAccount)
+//     );
+//     return true;
+//   } catch (e) {
+//     return false;
+//   }
+// }
```

Comment:
some code commented, where did this go then? (just a PR comment to remind we will need to either remove the commented code or figure out how to put it back)

Reply:
I remove this method from *coin-framework*. I leave its declaration inside *live-common*